dgg-utilities.user.js

```diff
@@ -2321,6 +2321,11 @@ function injectScript() {
       let resultNukes;
       let result;
 
+      // exclude whispers sent using slash commands or when the main chatbox is inactive
+      if (text.startsWith("/w ") || text.startsWith("/whisper ") || !$('#chat-windows-select span[title="Destiny GG"].active').length){
```

Comment:
I dunno how I feel about looking up a DOM element every time we type something, seems inefficient (maybe it's not, im just assuming, if you can do some kind of benchmark that would be nice). Maybe there's some way to do it with a mutation observer? Point it to some element to check what tab we're in or something, and change some boolean to false every time we switch to a DM tab.